src/entrypoints/surveys.ts

```diff
@@ -1,8 +1,11 @@
 import { generateSurveys } from '../extensions/surveys'
 
 import { window } from '../utils/globals'
+import { canActivateRepeatedly } from '../extensions/surveys/surveys-utils'
 
 if (window) {
+    ;(window as any).__PosthogExtensions__ = (window as any).__PosthogExtensions__ || {}
+    ;(window as any).__PosthogExtensions__.canActivateRepeatedly = canActivateRepeatedly
```

Comment:
maybe this should even be __PosthogExtensions__.__Surveys__ but 🤷 

but regardless this is the mechanism we use to push code out of the main bundle so that folk who don't use surveys (or replay or web vitals etc etc) don't have to download the code for that feature

accessing `canActivateRepeatedly` causes the bundler to pull in preact and you only need that with surveys enabled